.vitepress/theme/Post.vue

```diff
@@ -18,33 +18,43 @@ const prevPost = computed(() => posts[findCurrentIndex() + 1]);
   <article class="prose">
     <header class="mb-8">
       <Date :created-date="date" :updated-date="route.data.lastUpdated" />
-      <h1 class="!m-0">{{ $frontmatter.title }}</h1>
-      <dl class="flex text-sm font-medium leading-5 whitespace-nowrap">
+      <h1 class="!m-0 text-gray-400">{{ $frontmatter.title }}</h1>
+      <dl class="flex text-sm font-medium leading-10 whitespace-nowrap">
         <dt class="sr-only">Name</dt>
-        <dd class="text-gray-900 whitespace-nowrap">{{ $frontmatter.author }}</dd>
+        <dd class="text-gray-900 whitespace-nowrap dark:text-gray-400">
+          {{ $frontmatter.author }}
+        </dd>
         <dt v-if="$frontmatter.twitter" class="sr-only">Twitter</dt>
         <dd v-if="$frontmatter.twitter">
           <span class="px-2">|</span>
           <a
             :href="'https://twitter.com/' + $frontmatter.twitter"
             target="_blank"
             rel="noopnener noreferrer"
+            class="dark:text-gray-100"
             >{{ $frontmatter.twitter }}</a
           >
         </dd>
       </dl>
     </header>
 
-    <Content class="mb-10" />
+    <Content class="mb-10 dark:text-gray-400 leading-1 tracking-tight" />
```

Comment:
@ChrisShank I can't seem to work out how to affect the colors inside this tag

Reply:
Hmmm… i can look into it tomorrow. I was waiting to implement dark theme until tailwind typography supported it (which should have been recently)

Reply:
yeah I saw the PR come through https://github.com/tailwindlabs/tailwindcss-typography/pull/141/files